terraform/Yandex Cloud/main.tf

```diff
@@ -0,0 +1,42 @@
+terraform {
+  required_providers {
+    yandex = {
+      source = "yandex-cloud/yandex"
+    }
+  }
+  required_version = ">= 0.13"
+}
+
+provider "yandex" {
+  # Don't forget to set YC_TOKEN var for authentication
```

Comment:
instead, you might make an uninitialized variable in `*.tf` and initialize it using `*.tfvars` file ![](https://steamcommunity-a.akamaihd.net/economy/emoticon/:ohh_yeah:)